vncviewer/DesktopWindow.cxx

```diff
@@ -322,6 +324,29 @@ void DesktopWindow::setCursor(int width, int height,
 }
 
 
+void DesktopWindow::setCursorPos(const rfb::Point& pos)
+{
+  if (!mouseGrabbed) {
+    // Do nothing if we do not have the mouse captured.
+    return;
+  }
```

Comment:
We probably shouldn't do this if we aren't focused as well.

Reply:
wouldn't that make it hard to exit the client? in any case, moved to #1212.

Reply:
How so? The warping should generally be in the way in such cases, not helping?